packages/streamline/src/components/buttons/button-dock/button-dock.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react-native';
+import { ButtonDock } from './button-dock';
+import { renderWithProvider } from '../../../testing/render-with-provider';
+import { ButtonDockProps } from './button-dock.type';
+
+const props: ButtonDockProps = {
+  primaryButton: {
+    onPress: jest.fn(),
+    text: 'Primary Button',
+    appearance: 'primary',
+  },
+  secondaryButton: {
```

Comment:
```suggestion
  secondary: {
```
Nitpick: can we remove button suffix on primary & secondary? As we are already in a ButtonDock